lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -261,6 +261,10 @@ class CkImage implements ui.Image, StackTraceDebugger {
   @override
   CkImage clone() {
     assert(_debugCheckIsNotDisposed());
+    if (videoFrame != null) {
+      final VideoFrame videoFrameClone = videoFrame!.clone();
+      return CkImage.cloneOf(box, videoFrame: videoFrameClone);
+    }
     return CkImage.cloneOf(box);
```

Comment:
I think the above 5 lines can be compressed into one:

```dart
return CkImage.cloneOf(box, videoFrame: videoFrame?.clone());
```

(bonus: also remove a null assertion)

Reply:
Done.